test/cases/imds-errors.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Should there be a happy case?

Reply:
It's covered in `imds-token-refresh.sh`, though indirectly. I'll add one here.